tools/api/generate_go_protobuf.py

```diff
@@ -121,16 +121,24 @@ def updated(repo):
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Generate Go protobuf files and sync with go-control-plane')
+    parser.add_argument('--skip_sync', action='store_true')
```

Comment:
how about reversing this - ie `--sync` and `store_false`

Reply:
Yeah, that works.  I've made the change locally.  Would you prefer a commit on top, or re-writing the original commit. I'm not 100% sure what the policy is here

Reply:
once there are reviewers, its prefered not to rebase/force - sometimes personally i find it necessary, but best avoided

Reply:
Ack.  Extra commit pushed! 👍